### ISSUE TITLE
Missing accessibility features

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#38]
Hi there! I wanted to kindly point out that it seems like some accessibility features might be missing in the project. Including these would greatly improve the experience for all users. Thanks for considering this!
